src/console/dlt-control.c

```diff
@@ -488,6 +488,7 @@ int main(int argc, char* argv[])
             return -1;
         }
     }
```

Comment:
define based check is not added in all the places where serial is used in dlt-control. In other test applications it is added but it is missed here. please update.